src/modules/shared/components/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,342 @@
+import { render, screen } from '@testing-library/react';
+
+import Footer from './Footer';
+import { footerType } from './Footer.constants';
+
+const feedbackMock = jest.fn();
+
+// Default
+
+describe('Component: <Footer /> (default)', () => {
+	beforeEach(() => {
+		render(
+			<Footer
+				type={footerType.default}
+				links={[
```

Comment:
Zet verschillende mock data onder `/Footer/__mocks__/index.ts` dan kan je van daaruit importen voor uw tests, stories en eventueel als dummy data in de app.